src/api-binder.ts

```diff
@@ -460,7 +460,18 @@ export class APIBinder {
 
 		await Bluebird.resolve(
 			this.sendReportPatch(stateDiff, { apiEndpoint, uuid }),
-		).timeout(conf.apiTimeout);
+		)
+			.catch(StatusError, (e: StatusError) => {
```

Comment:
This would have made sense to me as a normal `try/catch` as we're in an `await`

Reply:
Maybe, but we had to assign the timeout anyway, and the predicate argument of bluebird's catch can make things a bit nicer.

Reply:
I know that's why we have the bluebird wrapper, and whilst the sugar is nice it does deviate from what is done everywhere else in the (TS) codebase so I'd go with consistency for `.then`/`.catch` style (ie try/catch instead of .then/.catch)